portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,27 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <meta charset="utf-8">
+    <script src="https://use.fontawesome.com/d1341f9b7a.js"></script>
     <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
+    <title>My Portfolio</title>
   </head>
   <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
-    </div>
-  </body>
+<div class="box">
+  <img src="images/beach2.0.png">
+  <h1>Welcome to My Portfolio</h1>
+    <h5>There's no formula to Life!!</h5>
+    <p>
+    Hey there, nice to meet you!!!. Click below for the blog.</a>
+    </p>
+    <a href="./blog.html" <button type="button" onclick="./blog.html">Blog!</button></a>
```

Comment:
This line is malformed. There is a missing `>` for the `<a` tag, with the result that the `button` tag is not being respected.

I think this should be either:

```
<a href="./blog.html">Blog!</a>
```

or

```
<button type="button" onclick="./blog.html">Blog!</button>
```